lib/transifex/resource.rb

```diff
@@ -23,5 +23,12 @@ def stats(lang)
       stats = client.get("/project/#{@project_slug}/resource/#{@slug}/stats/#{lang}")
       Transifex::Stats.new(stats).tap {|r| r.client = client }
     end
+
+    def update_content(content)
+      client.put(
```

Comment:
I believe they haven't defined the method `put` for the client.

Benedikt has implemented it like this: https://github.com/freeletics/nutrition-backend-rails/commit/6f691d2a2e40fab689ba8eee738c30b64c813c71#diff-0847b9ad5fc4481c53681608f48b05bdR20